### PR TITLE
Synthesis

### PR DESCRIPTION
**Pull request by william-d-smith**

### CODE REVIEW

File: tools/verdict-back-ends/verdict-bundle-parent/verdict-attack-defense-collector/src/main/java/com/ge/verdict/attackdefensecollector/Triple.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.ge.verdict.attackdefensecollector;
+
+import java.util.Objects;
+
+/**
+ * A generic pair. Correctly implements equals() and hashCode(), so may be used as a key to a hash
+ * map. Note that the L and R types must themselves correctly implement equals() and hashCode() for
+ * this to work correctly.
+ *
```
Comment (tuxji) [inline tools/verdict-back-ends/verdict-bundle-parent/verdict-attack-defense-collector/src/main/java/com/ge/verdict/attackdefensecollector/Triple.java:9]: Javadoc comes from Pair and should be updated for this Triple class.

File: tools/verdict-back-ends/STEM/Output/CompDep.csv [hunk 2]
```diff
@@ -0,0 +1,89 @@
+Comp,InputPort,InputCIA,OutputPort,OutputCIA
```
[the remark this answers is not in the document]
Comment (baoluomeng) [reply tools/verdict-back-ends/STEM/Output/CompDep.csv:1]: Those files were produced by Paul S. I think. I hope they won't break the testing suite of STEM runner.

File: tools/verdict-back-ends/verdict-bundle-parent/verdict-attack-defense-collector/src/main/java/com/ge/verdict/attackdefensecollector/Triple.java (hunk 1 above)
Comment (william-d-smith) [reply tools/verdict-back-ends/verdict-bundle-parent/verdict-attack-defense-collector/src/main/java/com/ge/verdict/attackdefensecollector/Triple.java:9]: fixed in a7b9701

**Comment by baoluomeng** [comment]
Merge the synthesis branch into master

**Pull request merged**